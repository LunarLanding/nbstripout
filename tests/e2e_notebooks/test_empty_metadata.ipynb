{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "5c42035d",
   "metadata": {},
   "source": [
    "This notebook is missing `metadata.kernelspec`."
   ]
  }
 ],
 "metadata": {},
 "nbformat": 4,
 "nbformat_minor": 5
}
