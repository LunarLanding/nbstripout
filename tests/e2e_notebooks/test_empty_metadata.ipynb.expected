{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "0",
   "metadata": {},
   "source": [
    "This notebook is missing `metadata.kernelspec`."
   ]
  }
 ],
 "metadata": {},
 "nbformat": 4,
 "nbformat_minor": 5
}
